spec/features/reviewing_step_by_steps_spec.rb

```diff
@@ -55,6 +69,24 @@ def when_I_view_the_step_by_step_page
     visit step_by_step_page_path(@step_by_step_page)
   end
 
+  def then_the_page_should_have_title_caption(caption)
```

Comment:
I'd suggest that you minimise the arguments that are passed through to the test steps. I'm wondering if this was to be done to help with reusability? 

With this style of test I'd suggest that steps should not be optimised for re-usability. I think the steps should be very tailored to the situation for maximum readability and if needs be a step can call a helper method. 

Apologies if my opinions here conflict with conventions deeply rooted in this app, but thought I'd bring it up as it looks like this app is subtly drifting from the more common pattern. Happy to chat about this.

Reply:
We've spoken IRL - I think this particular method is actually more readable with the argument passing, but agree the scenario itself is now quite long and that some steps should be combined 👍